mariadb/run.sh

```diff
@@ -1,12 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bashio
 set -e
 
-CONFIG_PATH=/data/options.json
 MARIADB_DATA=/data/databases
 
-DATABASES=$(jq --raw-output ".databases[]" $CONFIG_PATH)
-LOGINS=$(jq --raw-output '.logins | length' $CONFIG_PATH)
-RIGHTS=$(jq --raw-output '.rights | length' $CONFIG_PATH)
+DATABASES=$(bashio::config "databases[]")
```

Comment:
```suggestion
DATABASES=$(bashio::config "databases")
```

Bashio will automatically return an Bash array in case it is a list.

Secondly, this can be placed directly in the for loop below, removing the need for this variable at this point.